src/main/java/com/mycompany/exercisetwo/Cow.java

```diff
@@ -0,0 +1,47 @@
+package com.mycompany.exercisetwo;
+
+import java.time.LocalDate;
+import java.time.Period;
+
+/**
+ *
+ * @author Cristian Gallego
+ */
+public class Cow extends Mammalian implements Death {
+
+    public String result;
```

Comment:
You do not have to keep this variable as global reference you can create it inside die method

Reply:
Got it. Changed to a local scope.